descwl_shear_sims/sim.py

```diff
@@ -55,6 +54,7 @@ def make_sim(
     cosmic_rays=False,
     bad_columns=False,
     star_bleeds=False,
+    galsim_rng=None,
```

Comment:
Setting this to `None` will cause the seed to differ each time and make the sim not reproducible. Instead, we need to draw a seed from the input numpy RNG and instantiate the galsim_rng internally.

Reply:
That is how it is suppose to be done. The thing is, it is not used by the original script, so to not change it I did this "trick" to avoid having an error. What I would suggest is to also use the galsim noise generator even on the original test so this can be propagated correctly. That would mean changing [this line](https://github.com/LSSTDESC/descwl-shear-sims/blob/02fd39dbe4c982c575193ea861bbaa51b99466f0/descwl_shear_sims/sim.py#L333).

Reply:
I'm not being clear. We don't use the galsim RNG infrastructure in this sim package. So the RNG instantiation has to be changed. We cannot merge this code in its current form.

Reply:
What I mean is that this is not use by the original purest script and it is correctly instantiate in the one I have done where it is used so normally it should not be a problem. But I agree on the fact that this looks ugly at the moment.